README.md

```diff
@@ -1,33 +1,222 @@
+
+![Fig. Virtual Catalog](docs/media/screen-virtual-catalog.png)
+
+#### Scenarios 
+
+[Add Virtual Catalog](/docs/add-virtual-catalog.md)
+
+### Categories
+
+A category is a container for other categories or items. Category in the catalog can have sub-categories.
+
+Categories allow building hierarchy and relationship between various items in the catalog. This helps to make it easier for customers to navigate on client side to the items they would like to purchase.
+
+Large catalogs containing many items might be reasonable to organize using parent-child relations.
+
+![Fig. Categories](docs/media/screen-categories.png)
+
+#### Scenarios
+
+[Manage Categories](/docs/manage-categories.md)
+
+### Items (Product)
+
+A product with its variations in the system is presented as a collection of products (variations).
+
+There are two types of products presented in the Catalog- physical products and digital products.
+
+The title (main) product has variative property set of values. And all other products (variations) has the relation with the title product. Every product that has relation to the title product (variation) can inherit some property values of the title product (description, assets etc.) but can also override the values of those properties if they differ from the title product (variation) property values.
+
+To show such a product with variations on the storefront, the title product (main) is loaded with all the related products (variations). Property values of the title product are used for product description. SKU selector for such a product with variations is built using variations properties of the title product and its variations.
+So, an item concept in Virto Commerce is a generalization of Variation/SKU or Product. An item corresponds to item in the store that can be promoted and sold.
+
+![Fig. Product Variations](docs/media/screen-product-variations.png)
+
+User can manage simple item properties as well as complex information:
+
+1. Custom properties.
+1. SEO information.
+1. Associations (associated items).
+1. Assets.
+1. Images.
+1. Variations (SKUs).
+1. Inventory
+1. Prices.
+
+Inventory and prices management are integrated from other modules - “Virto Commerce Inventory” and “Virto Commerce Pricing” respectively.
+
+![Fig. Physical Item Details](docs/media/screen-physical-item-details.png)
+
+#### Scenarios
+
+[Manage Products](/docs/manage-physical-products.md)
+
+[Product Indexing](/docs/product-indexing.md)
+
+### Variations
+
+In case there is a need to add variations of the product, a relation should be implemented between the products as well as special type of variative properties definition added.
+
+Thus, a product with its variations in the system is presented as a collection of products (variations). The title (main) product can be also a variation and can have variative property values set. And all other products (variations) has the relation with the title product.
```

Comment:
It's taken from the document published on our we site. Yes, it means "And all other products (variations) has the relation with the title product."

Reply:
"The title (main) product can be also a variation and can have variative property values set. And all other products (variations) has the relation with the title product."

I see some confusion here. Using this description I do not see clear difference between main product and its variations. AFAIK there is the difference.
1. Main product cannot be the variation of another product (maybe I'm wrong there), where as "The title (main) product can be also a variation" tells the opposite.
2. Cannot fully understand this phrase - "And all other products (variations) has the relation with the title product". Does it mean all product entities in general - "We have 2 types of products - title products and variations, which are related to the title ones."? Or does it mean variations for one product - "Product variations have the relation with the title product."?

Reply:
**Main product cannot be the variation of another product (maybe I'm wrong there), where as "The title (main) product can be also a variation" tells the opposite.**
As I mentioned before, this was taken from the document published on out web site. For me it also sounded confusing. So, does it mean there is a mistake in that document? According to what I see in the system, the main product can not be a variation of another product. So, I think we need to remove this sentence from our document.

Reply:
Yes, it seems like a mistake to me. I would ask @OlegoO on details.

Reply:
the main products is variant too because he has SKU and Variation properties.
The title (main) product can be also a variation and can have variative property values set.

Please look catalog on demo site
![image](https://user-images.githubusercontent.com/7639413/60086996-6f246980-973c-11e9-890c-ff4670e30c6f.png)

Reply:
@mayaalsamarrai  @OlegoO Got the idea of this sentence.
Both title products and variations are kind of products. The only difference between them (and in implementation details) is that title product have no parent main product, whereas variations have parent product. So we need to tell that title product is a parent variation (not can be, seems this make a confusion to me) to which all other variation relate as a children.

"Thus, a product with its variations in the system is presented as a collection of products (variations). The title (main) product can be also a variation and can have variative property values set. And all other products (variations) has the relation with the title product."

To clarify, lets rephrase it as:
"Thus, a product with its variations in the system is presented as a collection of products (variations). The title (main) product **is a parent** variation and **has all variation properties**. And all other variations are related as children to the title product."

What do you think?

Reply:
Now it's clear enough! Thank you!